Assets/Scripts/Audio/AudioManager/MechanicAudioManager.cs

```diff
@@ -47,15 +48,15 @@ public class MechanicAudioManager : AbstractAudioManager {
     }
 
     public void playButtonSound() {
-        playOneShotSound(buttonClip, button, buttonVolume);
+		playOneShotSound(instance.buttonClip, instance.button, buttonVolume);
```

Comment:
apparently yea, so if you don't have instance.bla, it will try to use "this", which is destroyed if you already have a singleton instance around
Learn it the hard way..